### ISSUE TITLE
MangaBat: Blocked by CloudFlare

### ISSUE THREAD

**Issue by akshar2324** [keiyoushi/tachiyomi-extensions-source#6877]
### Source information

Mangabat 1.4.10

### Source language

English

### Steps to reproduce

I can't stay step i was reading manga but suddenly i got retry error and when i open webview it said i am blocked 
![Screenshot_20241228_210447_app_mihon_ReaderActivity](https://github.com/user-attachments/assets/e781fa6b-44d2-4907-b78d-b0690b83c3e8)
![Screenshot_20241228_210611_app_mihon_WebViewActivity](https://github.com/user-attachments/assets/836c1b60-4500-45b9-817a-a20d6ae638fc)

### Expected behavior

Chapter should show images

### Actual behavior

I am getting same when i use open in browser

### Mihon/Tachiyomi version

0.17.1

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by akshar2324** [comment]
After some testing other manga are working from same source but

https://readmangabat.com/read-yt402938


I am getting error in this one

**Comment by Akshar062** [comment]
> After some testing other manga are working from same source but
> 
> https://readmangabat.com/read-yt402938
> 
> 
> I am getting error in this one 

https://v7.mkklcdnv6tempv3.com/img/tab_27/04/56/92/to997049/chapter_24/3-1714948065-o.jpg

**Comment by Norsze** [comment]
Actually, this is an issue of image server 1 on the source not having the chapter, try opening chapter 24 of https://readmangabat.com/read-yt402938 on the website, it is broken. I don't know if image server 2 can be requested as a feature request or smth. But in general this is the site that is broken and we can't fix it.
